packages/utils/projen-blueprint/src/blueprint.ts

```diff
@@ -102,6 +103,10 @@ export class ProjenBlueprint extends typescript.TypeScriptProject {
     };
     super(finalOpts);
 
+    this.addFields({
+      homepage: 'https//aws.amazon.com',
```

Comment:
I think we should add `homepage` as an optional parameter to the `ProjenBlueprintOptions` interface.

If the value is provided, then we add the `homepage` field to the `package.json`. If not, we do nothing (the current behavior). Otherwise, we'll start setting the homepage of all custom blueprints to this URL until your next PR to set this to the project URL in `blueprint-builder`.

Also, looks like the typo in the URL originates here: `https//aws.amazon.com`